corehq/apps/reports/models.py

```diff
@@ -644,6 +644,12 @@ class FormExportSchema(HQExportSchema):
     def update_schema(self):
         super(FormExportSchema, self).update_schema()
         self.update_question_schema()
+        # update SplitColumn options
+        for column in [column for table in self.tables for column in table.columns]:
+            if isinstance(column, SplitColumn):
+                question = self.question_schema.question_schema.get(column.index)
+                column.options = question.options
+                column.ignore_extras = True
```

Comment:
column is local to this `FormExportSchema` object. This doesn't get saved to the DB either but even if it did it wouldn't matter.